src/test/scala/uk/gov/hmrc/api/models/constants/JsonPaths.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.api.models.constants
+
+object JsonPaths {
+  val authType: String       = "authType"
+  val code: String           = "code"
+  val eoris: String          = "eoris"
+  val errorDetail: String    = "errorDetail"
+  val errors: String         = "errors"
+  val message: String        = "message"
+  val path: String           = "path"
+  val processingDate: String = "processingDate"
```

Comment:
Is this in use?

Reply:
Some of them are, I'll remove the unneeded ones